Sources/MapboxDirections/RouteOptions.swift

```diff
@@ -123,31 +123,31 @@ open class RouteOptions: DirectionsOptions {
     open var roadClassesToAvoid: RoadClasses = []
     
     /**
-     A number that influences whether the route should prefer or avoid alleys or narrow service roads between buildings.
+     An optional that represents the number that influences whether the route should prefer or avoid alleys or narrow service roads between buildings.
      
-     This property has no effect unless the profile identifier is set to `DirectionsProfileIdentifier.automobile` or `DirectionsProfileIdentifier.walking`.
+     This property has no effect unless it is explicitly set as it defaults to nil, and works best when the profile identifier is set to `DirectionsProfileIdentifier.automobile` or `DirectionsProfileIdentifier.walking`.
```

Comment:
The [Directions API documentation](https://docs.mapbox.com/api/navigation/directions/#retrieve-directions) doesn’t explicitly restrict this option to any particular profile, even if it happens to be unsupported in the cycling profile, so I think we can remote this note entirely. It’s probably unnecessary to state that an unset property is set to `nil` and therefore has no effect, since no default value is present in the property signature.